packages/xo-web/src/xo-app/vm-import/ova/index.js

```diff
@@ -45,10 +39,10 @@ const RESOURCE_TYPE_TO_HANDLER = {
   '17': (
     { disks },
     {
-      'rasd:AddressOnParent': position,
-      'rasd:Description': description = 'No description',
-      'rasd:ElementName': name,
-      'rasd:HostResource': resource,
+      AddressOnParent: position,
+      Description: description = 'No description',
+      ElementName: name,
+      HostResource: resource,
     }
   ) => {
     const diskId = resource.match(/^(?:ovf:)?\/disk\/(.+)$/)
```

Comment:
Doesn't the regex need to be changed?

Reply:
no, there still might or might not be a prefix there, it's the text of a node.
```
<rasd:HostResource>ovf:/disk/vmdisk1</rasd:HostResource>
```